reactFirstProject/src/App.jsx

```diff
@@ -1,13 +1,53 @@
-import { useState } from 'react'
-import './App.css'
+let states = [
+  {
+    id:0,
+    name: "Quiz #1",
+    questions:5,
+    score:"",
```

Comment:
```suggestion
    score: null,
```
it is unlikely that a variable that is supposed to be numeric will suddenly become a string. For empty values - use null. It is more realistic